lib/client.js

```diff
@@ -22,7 +22,11 @@ var clientOptions = require('./client-options');
  *  Only one contact point is required (the driver will retrieve the address of the other nodes automatically),
  *  but it is usually a good idea to provide more than one contact point, because if that single contact point is unavailable, the driver will not be able to initialize correctly.
  * </p>
- * @property {String} keyspace
+ * @property {String} keyspace The logged keyspace for all the connections created within the {@link Client} instance.
+ * @property {Number} refreshSchemaDelay The default window size in milliseconds used to debounce node list and schema
+ * refresh metadata requests. Default: 1000.
+ * @property {Boolean} isMetadataEnabled Determines whether client-side schema metadata retrieval and update is enabled.
```

Comment:
With respect to my comment about 'request schema metadata on demand' i don't think that will work because the driver will currently only fetch the table if it knows the keyspace is there, which is wouldn't since the keyspace wouldn't be known (although right now it will since the metadata is grabbed on control connection).

Also it currently wouldn't be possible to know if a table is stale in the local metadata or not, so that approach wouldn't work either.

I think the best route would be for `isMetadataEnabled` to never fetch schema metadata (which means not building token map either).